### ISSUE TITLE
Incorrect/Mismatched Game IDs in NBA API functions

### ISSUE THREAD

**Issue by mattmattoni** [sportsdataverse/hoopR#135]
Awesome package. However, there is an issue that there is a mismatch between game IDs across functions. 

For example, if I use the load_nba_team_box(2024) function, the 12/4/2023 game between the Pelicans and Kings has game ID 401607492. However, this Game ID is not a valid input to any other arguments using the NBA API. Additionally, it does not follow the NBA API Game ID format, and does not even have the correct number of digits. 
Any NBA API function will give the following error:
> nba_boxscoreadvancedv2(401607492)
Error in nba_boxscoreadvancedv2(401607492) : object 'df_list' not found

Is there an error in game ID acquisition? If not, is there a way to easily find Game IDs that match the NBA's API? Or, is there some way to convert between IDs?

**Comment by saiemgilani** [comment]
Yeah, the loading functions currently use ESPN `game_id`s and to find the NBA API's `game_id`'s, use the [`nba_schedule()`](https://hoopr.sportsdataverse.org/reference/nba_schedule.html) function, which uses the NBA API as its data source. See the [getting started vignette](https://hoopr.sportsdataverse.org/articles/getting-started-hoopR.html#the-data) if you're confused about which functions correspond to others. I suppose the load functions aren't perfectly clearly documented

**Comment by saiemgilani** [comment]
The asked-for mapping now exists: the ESPN↔NBA-Stats crosswalks shipped as `load_nba_player_crosswalk()` / `load_nba_team_crosswalk()` / `load_nba_schedule_crosswalk()` (397078fb4, backed by the published nba_crosswalk releases). The id-convention difference between the loaders (ESPN) and nba_* functions (NBA Stats) is by design; the crosswalk joins them. Closing.
